examples/gno.land/p/demo/ufmt/ufmt.gno

```diff
@@ -17,16 +18,20 @@ func Println(args ...interface{}) {
 		switch v := arg.(type) {
 		case string:
 			strs = append(strs, v)
+		case (interface{ String() string }):
```

Comment:
Since this is being re-used, create a `stringer` type.

Reply:
Hey, Morgan. 
If you don't mind I prefer to keep this one as is.

The test `ufmt_test.gno` already has this:

```
package ufmt

import (
	"errors"
	"fmt"
	"testing"
)

type stringer struct{}

func (stringer) String() string {
	return "I'm a stringer"
}

func TestSprintf(t *testing.T) {
```

Also I think it's more direct for people browsing to this package (especially beginners) to see something like `case (interface{ String() string }):`

I will apply all your other suggestions however.
Thank you for reviewing this, let me know if any more problem.